cmd/clusterctl/cmd/root.go

```diff
@@ -20,45 +20,100 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/MakeNowJust/heredoc"
 	"github.com/spf13/cobra"
-	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/klog"
 )
 
-const deprecationMsg string = "NOTICE: clusterctl has been deprecated in v1alpha2 and will be removed in a future version."
-
-var helpTemplate = fmt.Sprintf(`%s
-{{with (or .Long .Short)}}{{. | trimTrailingWhitespaces}}
-
-{{end}}{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`, deprecationMsg)
+var cfgFile string
 
 var RootCmd = &cobra.Command{
 	Use:   "clusterctl",
-	Short: "cluster management",
-	Long:  `Simple kubernetes cluster management`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		return cmd.Help()
-	},
+	Short: "clusterctl controls a management cluster for Cluster API",
+	Long: LongDesc(`
+		Get started with Cluster API using clusterctl for initializing a management cluster and for creating yaml
```

Comment:
Should the long description here also mention that it is used for managing provider deployments as well?

Reply:
Done, let me know if it is more clear now.